service/src/main/java/com/ericsson/eiffel/remrem/generate/controller/RemremGenerateController.java

```diff
@@ -141,7 +145,7 @@ private JsonObject erLookup(final JsonObject bodyJson, Boolean failIfMultipleFou
 
                     // prepare ER Query
                     String Query = ERLookupController.getQueryfromLookup(lookupLinks.get(i).getAsJsonObject());
-                    String url = erlookupConfig.getErURL() + Query;
+                    String url = erlookupConfig.getErURL() + Query + String.format("&shallow=%s&pageSize=%d",connectToExternalERs, limit);
```

Comment:
connectToExternalERs=true means shallow=false and vice versa, so this value needs to be negated.

Reply:
Ok, so what about the default value, as per requirement default value should be true.

Reply:
It should be consistent over the whole documentation. Preferably the default value should only be mentioned in one place, and linked from other places.

Reply:
We concluded like Default values for lookupInExternalERs is False and lookupLimit is 1
If lookupInExternalERs =false its acts as shallow=true and if lookupInExternalERs= true then shallow =false